server/src/test/java/org/apache/iotdb/db/integration/IoTDBMetadataFetchIT.java

```diff
@@ -201,6 +202,29 @@ public void databaseMetaDataTest() throws ClassNotFoundException, SQLException {
     }
   }
 
+  @Test
+  public void showVersion() throws SQLException, ClassNotFoundException {
+    Class.forName(Config.JDBC_DRIVER_NAME);
+    try (Connection connection = DriverManager
+        .getConnection(Config.IOTDB_URL_PREFIX + "127.0.0.1:6667/", "root", "root");
+        Statement statement = connection.createStatement()) {
+      String sql = "show version";
+      try {
+        boolean hasResultSet = statement.execute(sql);
+        if(hasResultSet) {
+          try(ResultSet resultSet = statement.getResultSet()) {
+            ResultSetMetaData resultSetMetaData = resultSet.getMetaData();
+            Assert.assertEquals(resultSetMetaData.getColumnLabel(1), IoTDBConstant.VERSION);
+          }
+        }
+      } catch (Exception e) {
+        fail(e.getMessage());
+      } finally {
+        connection.close();
```

Comment:
This call is unnecessary since you have put `connection` to try resource control.

Reply:
You mean to delete `connect.close()`.

Reply:
Yes